model/users_stats.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package model
+
+import (
+	"encoding/json"
+	"io"
+)
+
+type UsersStats struct {
+	TotalUsersCount int64 `json:"total_user_count"`
```

Comment:
The field name is `TotalUsersCount` but the JSON field is `total_user_count` ("user" vs "users"). Mind making the pluralization match?